src/ontodia/diagram/layoutData.ts

```diff
@@ -29,7 +31,7 @@ export interface LayoutLink {
 
 const serializedCellProperties = [
     'id', 'type',                              // common properties
-    'size', 'angle', 'isExpanded', 'position', // element properties
+    'size', 'angle', 'isExpanded', 'position', 'iri', 'group', // element properties
```

Comment:
You should also modify `normalizeImportedCell` to keep compatibility with previously saved diagrams.